Directory.Build.props

```diff
@@ -164,7 +164,6 @@
 
   <!-- Set some shortcuts for more terse conditions in project files -->
   <PropertyGroup>
-    <TargetsNetCoreApp Condition="$(TargetGroup.StartsWith('netcoreapp'))">true</TargetsNetCoreApp>
```

Comment:
There was a least one project using this it is probably worth keeping it for consistency.

Reply:
It is still there, few lines below. For some reason it was there twice.